lib/octicons_react/README.md

```diff
@@ -62,6 +62,70 @@ export default () => (
 )
 ```
 
+### `aria-labelledby`
+
+You have the option of adding accessibility information to the icon with the
+[`aria-labelledby` attribute][aria-labelledby] via the `aria-labelledby` prop. Using aria-labelledby referencing the id values of the title element provides the accessible name.
+
+```js
+// Example usage
+import {PlusIcon} from '@primer/octicons-react'
+
+export default () => (
+  <button>
+    <PlusIcon aria-labelledby="title" title="Add new item" /> New
```

Comment:
This example comes across confusing as we have the `button` and have three ways in which an accessible name can be computed:

- `aria-labelledby`
- `title`
- The text content `New`

Unfortunately I think `aria-labelledby` doesn't have a solid story when used within the context of a `button`. Is there a good alternate example that we could use for this scenario? Also happy to remove it as an explicit example if there isn't one handy.